fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/database/entities/CookieEntity.java

```diff
@@ -21,9 +18,13 @@
 @ToString
 public class CookieEntity {
     @Id
-    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "cookie_generator")
+    @SequenceGenerator(name = "cookie_generator", sequenceName = "cookie_id_seq", allocationSize = 1)
```

Comment:
Using allocationSize 1 is a bad idea from a performance perspective as it is an entity-per-request class. Use 50 as sane default. See below

Reply:
done